tests/fixtures/RegistryPageTestContactExtra.yml

```diff
@@ -0,0 +1,11 @@
+SilverStripe\Registry\Tests\Stub\RegistryPageTestContactExtra:
+  jimmy:
+    FirstName: Jimmy
+    Surname: Sherson
+    RegistryPageID: =>SilverStripe\Registry\Tests\Stub\RegistryPageTestPage.contact-registrypage-extra
```

Comment:
This is a little risky, since this fixture doesn't exist in this file. Perhaps you could combine the two fixture files you've created into one so that this could be used in isolation in future?